_posts/fr/newsletters/2022-09-28-newsletter.md

```diff
@@ -0,0 +1,225 @@
+  Anthony Towns [a posté][towns bi] sur la liste de diffusion Bitcoin-Dev une
+  description de fork de Bitcoin Core sur laquelle il travaille qui pourrait seulement
+  s'appliquer par défaut sur [signet][topic signet] et qui appliquera des règles
+  pour les propositions de soft fork avec des spécifications et des implémentations
+  de haute qualité. Cela devrait permettre aux utilisateurs d'expérimenter plus
+  facilement la modification proposée, notamment en comparant directement les
+  modifications entre elles (ex. comparer [OP_CHECKTEMPLATEVERIFY][topic
+  op_checktemplateverify] à [SIGHASH_ANYPREVOUT][topic sighash_anyprevout]).
+  Towns prévoit également d'inclure les changements majeurs proposés à la politique
+  de relai des transactions (tel que [package relay][topic package relay]).
+
+    Towns cherche à obtenir des critiques constructives sur l'idée de cette nouvelle
+    mise en œuvre de tests, appelée [Bitcoin Inquisition][], ainsi qu'une
+    relecture des [pull requests][bi prs] ajoutées aux paramètres initiaux du
+    soft forks.
+
+## Selection de Q&R du Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] est une des principales places pour les
+contributeurs d'Optech pour trouver les réponses à leurs questions---ou lorsque nous
+avons quelques instants pour aider les utilisateurs curieux ou confus. Dans cette
+chronique mensuelle, nous mettons en avant certaines des questions et réponses les
+plus postées depuis notre dernière édition.*
+
+- **Fees  Ratecards:** Lisa Neigut [a posté][neigut ratecards]
+  sur la liste de diffusion de Lightning-Dev une porposition de *fee ratecards*
+  qui permettrait à un noeud d'anoncer quatre niveaux de frais de transactions.
+  Par exemple, un paiement transmis qui laisse à un canal moins de 25 % de sa
+  capacité sortante disponible pour transmettre les paiements suivants devra
+  payer proportionnellement plus qu'un paiement qui laisse 75 % de sa capacité
+  de sortie disponible.
+
+    Le développeur ZmnSCPxj [a décrit][zmnscpxj ratecards] une façon simple
+    d'utiliser les ratecards, "vous pouvez modéliser une carte tarifaire
+    comme quatre canaux distincts entre les deux mêmes nœuds, avec des coûts
+    différents pour chacun. Si le chemin au coût le plus bas échoue, il suffit
+    d'essayer une autre route qui peut qui peut avoir plus de sauts mais un coût
+    effectif plus faible, ou bien essayer la même canal à un coût plus élevé."
+
+    La proposition prévoit également des redevances négatives.  Par exemple,
+    une chaîne pourrait subventionner les paiements qui lui sont transmis
+    lorsqu'il a plus de 75 % de capacité sortante en ajoutant 1 satoshi pour
+    chaque tranche de 1 000 satoshis de valeur de paiement.  Les frais négatifs
+    pourraient être utilisés par les marchands pour inciter les autres à restaurer
+    la capacité entrante des canaux fréquemment utilisés pour recevoir des paiements.
+  
```

Comment:
Trailing space here. I pushed a commit to address. (you can see such errors locally running `make production`)